libvmi/driver/xen/libxs_wrapper.h

```diff
@@ -23,17 +23,17 @@
 #include <config.h>
 #include <dlfcn.h>
 
-#if HAVE_XENSTORE_H
+#if (HAVE_XENSTORE_H == 1)
 #include <xenstore.h>
-#elif HAVE_XS_H
+#elif (HAVE_XS_H == 1)
 #include <xs.h>
 #endif
 
 #include "libvmi.h"
 
 struct xen_instance;
 
-#ifndef HAVE_LIBXENSTORE
+#if (HAVE_LIBXENSTORE == 0)
```

Comment:
@tklengyel not sure if this is the best syntax to test a C preprocessor define, what do you think ?

Reply:
I don't think you need paranthesis